docs/mission.md

```diff
@@ -0,0 +1,47 @@
+
+As members of the Flax team, we spend a majority of our time on **community engagement**. To make sure what we build is delightful for our current and future users, we hold design discussions in the open. This helps the community share ideas and engage with Flax core developers directly, while we learn from each others' complementary experiences and integrate the feedback. We [actively engage](https://github.com/google/flax/issues/208) with users via GitHub issues, discussions and pull requests. We share [design notes](https://flax.readthedocs.io/en/latest/design_notes/linen_design_principles.html), [philosophy](https://flax.readthedocs.io/en/latest/philosophy.html), and [recommended best practices](https://flax.readthedocs.io/en/latest/howtos.html) with the community. We improve our APIs based on feedback, while supporting our users through any transition—take a look at how we did it for our recent ["Linen" rewrite](https://github.com/google/flax/tree/main/flax/linen), as an example.
+
+---
+
+## Growing the Flax Team
+
+JAX and Flax have been growing rapidly, both in open source and within Google, and we're looking to grow the Flax team. With thoughtful technical and organizational structures in place, we hope to facilitate a 10x growth of the Flax user base. Due to the flexibility of interop through pure functions, we can imagine a broader Flax effort, with a flourishing ecosystem of libraries, to serve both an increased number of users, as well as **entirely new forms of research**.
+
+We see all of our work stemming from our foundational goal: **to serve our users**. We are constantly developing a deeper understanding of our users' needs and challenges, and we do whatever it takes to supply our users with the best experience. We discuss challenges and possible solutions with our users, especially when the right choice isn't apparent -- and these discussions lead us to continuous improvements. These improvements take many forms, such as improved documentation, error messages, or new solutions to common problems.
+
+Does this mission excite you? If so, take a look at [our job posting](https://careers.google.com/jobs/results/116638751486026438-software-engineer-jax-and-flax-google-research/). And even if you're not sure if the job posted fits your background exactly, or if now isn't the best time, please do reach out to us at [join-flax@google.com](mailto:join-flax@google.com)  — we'd love to hear from you and get to know you better.
```

Comment:
Our job posting currently says only Amsterdam. So we should a clarification here, e.g.:

"If so, take a look at our job posting [despite what it says, we'd consider candidates in Amsterdam, Zurich or the Bay Area]"